admin_manual/configuration/server/occ_command.rst

```diff
@@ -451,23 +451,38 @@ To update an existing value,  set ``--update-only``:
      --type=boolean --update-only
   Value not updated, as it has not been set before.
 
-Note that in order to write a Boolean, float, or integer value to the configuration file, you need to specify the type on your command. 
-This applies only to the ``config:system:set`` command. The following values are known:
+Note that in order to write a boolean, float, JSON, or integer value to the configuration file, you need to specify the type on your command.
+This applies only to the ``config:system:set`` command.
+The following values are known:
 
 * ``boolean``
 * ``integer``
 * ``float``
+* ``json``
 * ``string`` (default)
 
-When you want to e.g., disable the maintenance mode run the following command:
+Examples
+^^^^^^^^
+
+Disable the maintenance mode:
 
 ::
 
-  sudo -u www-data php occ config:system:set maintenance --value=false 
+  sudo -u www-data php occ config:system:set maintenance \
+     --value=false
      --type=boolean
+
   ownCloud is in maintenance mode - no app have been loaded
   System config value maintenance set to boolean false
 
+Create the ``app_paths`` config setting (using a JSON payload):
+
+::
+
+   sudo -u www-data php occ config:system:set apps_paths \
+      --type=json \
+      --value='[{"path":"/home/phil/git/owncloud/core/apps","url":"apps","writable":1},{"path":"/home/phil/git/owncloud/core/apps2","url":"apps2","writable":1}]'
```

Comment:
Do we want the example to be straight from my development system, with ``/home/phil/git`` ?
Or make up some other "typical" path from ``/`` to where ownCloud might be installed?

Reply:
Proposal
``/var/www/owncloud/apps``
``/mnt/external/www/owncloud/apps-external``

The first path is the classical oC install path
The second path is set to a „visible“ external mount but its final directory name is important ``apps external`` as we use this one in all other locations. Some may have also located the data directoy under the mounted location (as I do)...